### PR TITLE
Added gcc-6-32bit.cmake

### PR DESCRIPTION
**Pull request by Nadav-Eyesight**
Just a toolchain I found myself needing. I don't know if there's anything else I need to add for the automation, the contributing file doesn't mention anything.

### CODE REVIEW

File: gcc-6-32bit.cmake [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+polly_init(
+    "gcc 6 / c++14 support / 32 bit"
+    "Unix Makefiles"
+)
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_common.cmake")
+
+include("${CMAKE_CURRENT_LIST_DIR}/compiler/gcc-6.cmake")
+include("${CMAKE_CURRENT_LIST_DIR}/flags/cxx14.cmake")
```
Comment (ruslo) [inline gcc-6-32bit.cmake]: gcc toolchains by default comes with C++11 (e.g. `gcc-7`), if you want to have C++14 please rename it to `gcc-6-32bit-cxx14`.

**Comment by Nadav-Eyesight** [comment]
Done. Editing the toolchain table should be a requirement in https://github.com/ruslo/polly/blob/master/CONTRIBUTING.md

File: compiler/gcc-6-32bit-cxx14.cmake [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2013-2017, Ruslan Baratov
```
Comment (ruslo) [inline compiler/gcc-6-32bit-cxx14.cmake]: Please change the name of the file to `compiler/gcc-6.cmake`

**Comment by Nadav-Eyesight** [comment]
I totally forgot to change the line `include("${CMAKE_CURRENT_LIST_DIR}/compiler/gcc-6.cmake")`, which sort of fixed itself now :smile:

**Pull request merged**